plugins/resource_management/config/policy.json

```diff
@@ -33,6 +35,8 @@
   "resource_management:project_resource_create_request": "rule:access_to_project",
   "resource_management:project_resource_new_package_request": "rule:access_to_project",
   "resource_management:project_resource_create_package_request": "rule:access_to_project",
+  "resource_management:project_resource_reduce_quota": "rule:access_to_project",
+  "resource_management:project_resource_confirm_reduce_quota": "rule:access_to_project",
```

Comment:
I think this should be `rule:project_admin`. Members should not be able to reduce their project's quota. On the other hand, it is kind of understandable that you chose this in symmetry to "request quota", which is open to everyone. Since we're going to be billing on quota, we should restrict all quota-changing actions to `rule:project_admin`.